modules/RTC/TraceablePeerConnection.js

```diff
@@ -2163,7 +2163,7 @@ TraceablePeerConnection.prototype.createAnswer = function(constraints) {
     if (browser.supportsRtpSender() && this.isSimulcastOn()) {
         const videoSender
             = this.peerconnection.getSenders().find(sender =>
-                sender.track.kind === 'video');
+                sender.track !== null && sender.track.kind === 'video');
```

Comment:
I'm still getting this error when using unified:

`TypeError: undefined is not an object (evaluating 'this.peerconnection.getSenders().find(function(e){return null!==e.track&&"video"===e.track.kind}).setParameters'`

I think we need to check for falseyness, not null.

Reply:
Hum, I tried a couple of things locally and I never get video when in unified plan.

Reply:
I'm trying to reproduce the error right now. I disabled video support in BrowserCapabilites for unified plan in Safari because even with the new sdp-interop Safari doesn't work flawlessly.

Reply:
We need additional check here. 
```js
// videosender is absent for `startmuted` here
const videosender = this.peerconnection.getSenders().find(sender => {
     return sender.track && sender.track.kind === 'video';
})

//... undefined here
videosender && videosender.setParameters({
    ... simulcastParams,
    transactionId: "" //have no idea what value should be here, but safari requires it
})
```

Reply:
Yeah, sorry for the noise. Let's ignore video on unified for the purposes of this PR.

This is the proper fix for that error:

```
diff --git a/modules/RTC/TraceablePeerConnection.js b/modules/RTC/TraceablePeerConnection.js
index e6bef13..3c6c87e 100644
--- a/modules/RTC/TraceablePeerConnection.js
+++ b/modules/RTC/TraceablePeerConnection.js
@@ -2160,10 +2160,12 @@ const _fixAnswerRFC4145Setup = function(offer, answer) {
 };
 
 TraceablePeerConnection.prototype.createAnswer = function(constraints) {
-    if (browser.supportsRtpSender() && this.isSimulcastOn()) {
+    // TODO: Apply this on Firerox only since it's the only one which only does
+    // simulcast with RID.
+    if (browser.isFirefox() && this.isSimulcastOn()) {
         const videoSender
             = this.peerconnection.getSenders().find(sender =>
-                sender.track !== null && sender.track.kind === 'video');
+                sender.track.kind === 'video');
         const simParams = {
             encodings: [
                 {

```

As you can see I went back to explicitly checking for Firefox. The problem is that while other browsers support RTPSender, that particular funcion applies only if RID simulcast is used, which we only support on Firefox, and in an experimental capacity.

Reply:
I updated the diff with this fix.